src/Projection/MySqlProjectionManager.php

```diff
@@ -0,0 +1,337 @@
+            $options[self::OPTION_CACHE_SIZE] ?? self::DEFAULT_CACHE_SIZE,
+            $options[self::OPTION_PERSIST_BLOCK_SIZE] ?? self::DEFAULT_PERSIST_BLOCK_SIZE,
+            $options[self::OPTION_SLEEP] ?? self::DEFAULT_SLEEP
+        );
+    }
+
+    public function createReadModelProjection(
+        string $name,
+        ReadModel $readModel,
+        array $options = []
+    ): ReadModelProjection {
+        return new PdoEventStoreReadModelProjection(
+            $this->eventStore,
+            $this->connection,
+            $name,
+            $readModel,
+            $this->eventStreamsTable,
+            $this->projectionsTable,
+            $options[self::OPTION_LOCK_TIMEOUT_MS] ?? self::DEFAULT_LOCK_TIMEOUT_MS,
+            $options[self::OPTION_PERSIST_BLOCK_SIZE] ?? self::DEFAULT_PERSIST_BLOCK_SIZE,
+            $options[self::OPTION_SLEEP] ?? self::DEFAULT_SLEEP
+        );
+    }
+
+    public function deleteProjection(string $name, bool $deleteEmittedEvents): void
+    {
+        $sql = <<<EOT
+UPDATE $this->projectionsTable SET status = ? WHERE name = ? LIMIT 1;
+EOT;
+
+        if ($deleteEmittedEvents) {
+            $status = ProjectionStatus::DELETING_INCL_EMITTED_EVENTS()->getValue();
+        } else {
+            $status = ProjectionStatus::DELETING()->getValue();
+        }
+
+        $statement = $this->connection->prepare($sql);
+        $statement->execute([
+            $status,
+            $name,
+        ]);
+    }
+
+    public function resetProjection(string $name): void
+    {
+        $sql = <<<EOT
+UPDATE $this->projectionsTable SET status = ? WHERE name = ? LIMIT 1;
+EOT;
+
+        $statement = $this->connection->prepare($sql);
+        $statement->execute([
+            ProjectionStatus::RESETTING()->getValue(),
+            $name,
+        ]);
+    }
+
+    public function stopProjection(string $name): void
+    {
+        $sql = <<<EOT
+UPDATE $this->projectionsTable SET status = ? WHERE name = ? LIMIT 1;
+EOT;
+
+        $statement = $this->connection->prepare($sql);
+        $statement->execute([
+            ProjectionStatus::STOPPING()->getValue(),
+            $name,
+        ]);
+    }
+
+    public function fetchProjectionNames(?string $filter, int $limit = 20, int $offset = 0): array
+    {
+        if (1 > $limit) {
+            throw new OutOfRangeException(
+                'Invalid limit "'.$limit.'" given. Must be greater than 0.'
+            );
+        }
+
+        if (0 > $offset) {
+            throw new OutOfRangeException(
+                'Invalid offset "'.$offset.'" given. Must be greater or equal than 0.'
+            );
+        }
+
+        $where = [];
+        $values = [];
+
+        if (null !== $filter) {
+            $where[] = '`name` = :filter ';
+            $values[':filter'] = $filter;
+        }
+
+        $whereCondition = implode(' AND ', $where);
+
+        if (! empty($whereCondition)) {
```

Comment:
$where should only get a value if $filter is set. So this condition could be combined with the later one.